launchable/commands/subset.py

```diff
@@ -409,6 +422,15 @@ def run(self):
 
                     res = client.request("post", "subset", timeout=timeout, payload=payload, compress=True)
 
+                    if res.status_code == 422:
+                        click.echo(
+                            click.style(
+                                "Subset creation failed. Error: {}".format(
+                                    res.json().get("reason")),
+                                fg="red"),
+                            err=True)
+                        sys.exit(1)
```

Comment:
Can we os.exit here? I guess we should do a fallback

Reply:
Ok, I made a necessary change as follows:

```
$ Users/ono-max/.local/share/virtualenvs/cli-o7XEfK99/bin/launchable  subset --target 80% --prioritized-tests-mapping 'sample.json' maven src/test/java

Error: Include invalid prioritized tests json in your request
422 Client Error:  for url: http://localhost:8080/intake/organizations/launchableinc/workspaces/mothership/subset
Warning: the service failed to subset. Falling back to running all tests
example.MulTest
example.AddTest
example.SubTest
example.DivTest
```

Reply:
Sorry, you don't have to do fallback in this case, because it's mapping file format error.

Reply:
[b1736cb](https://github.com/launchableinc/cli/pull/622/commits/b1736cb88f6307d75df23d7b879963e4d281c66d)